src/main/java/org/cryptomator/common/Environment.java

```diff
@@ -34,7 +34,10 @@ public class Environment {
 	private static final String PLUGIN_DIR_PROP_NAME = "cryptomator.pluginDir";
 	private static final String TRAY_ICON_PROP_NAME = "cryptomator.showTrayIcon";
 
-	private Environment() {}
+	private Environment() {
+		//hack, needed for logging directory to be setup correctly
+		PropertiesPreprocessor.run();
+	}
```

Comment:
This gets invoked after static initialization, so the Logger already exists at this point.